pkg/controller/replication/replication_controller.go

```diff
@@ -353,22 +353,24 @@ func (rm *ReplicationManager) updatePod(old, cur interface{}) {
 		return
 	}
 	curPod := cur.(*api.Pod)
-	rc := rm.getPodController(curPod)
-	if rc == nil {
-		return
-	}
 	oldPod := old.(*api.Pod)
-
+	glog.V(4).Infof("Pod %s updated %+v -> %+v.", curPod.Name, oldPod, curPod)
```

Comment:
%+v on a pod is really chatty, can we log something more specific? if we're only interested in the labels, maybe that's enough?